doc/source/ecosystem.rst

```diff
@@ -149,13 +149,36 @@ for pandas ``display.`` settings.
 qgrid is "an interactive grid for sorting and filtering
 DataFrames in IPython Notebook" built with SlickGrid.
 
-`Spyder <https://github.com/spyder-ide/spyder/>`__
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+`Spyder <https://www.spyder-ide.org/>`__
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Spyder is a cross-platform PyQt-based IDE combining the editing, analysis,
+debugging and profiling functionality of a software development tool with the
+data exploration, interactive execution, deep inspection and rich visualization
+capabilities of a scientific environment like MATLAB or Rstudio.
+
+Its `Variable Explorer <https://docs.spyder-ide.org/variableexplorer.html>`__
+allows users to view, manipulate and edit pandas Index, DateTimeIndex, Series,
```

Comment:
Small nits: 

- `DateTimeIndex` --> `DatetimeIndex `

- By TimeSeries does this mean Timestamp objects?  Pandas doesn't strictly have a `TimeSeries` object though is great for timeseries data.

Reply:
> DateTimeIndex --> DatetimeIndex

Thanks, fixed.

> By TimeSeries does this mean Timestamp objects? Pandas doesn't strictly have a TimeSeries object though is great for timeseries data.

Silly mistake on my part; I've been doing too much developing and not enough *using* Spyder and the PyData stack lately. I got it confused with DatetimeIndex, although Spyder does support editing Timestamps, periods, etc. as well as standard library ``datetime`` dates. Just removed it instead.